ggml-backend.c

```diff
@@ -1352,6 +1384,10 @@ static void sched_reset(ggml_backend_sched_t sched) {
     memset(sched->node_talloc,   0, sizeof(sched->node_talloc[0])   * hash_size);
     memset(sched->node_copies,   0, sizeof(sched->node_copies[0])   * hash_size);
 
+    // TODO: should we clear the callbacks?
+    //sched->callback_eval = NULL;
+    //sched->callback_eval_user_data = NULL;
+
```

Comment:
I think this is fine, we don't need to clear the callbacks here, the `reset` function is meant to prepare the sched for the next graph evaluation, resetting the allocators and the backend assignments (similar to `ggml_allocr_reset`).